test/runtests.jl

```diff
@@ -46,6 +46,14 @@ isordered(::Type{T}) where {T<:AbstractDict} = false
     callsites = find_callsites_by_ftt(iterate, (Base.IdSet{Any}, Union{}); optimize=false)
     @test callsites[1].info isa Cthulhu.ConstPropCallInfo
 
+    if Cthulhu.EFFECTS_ENABLED
+        effects = Cthulhu.get_effects(callsites[1].info)
+        @test !Core.Compiler.is_consistent(effects)
+        @test Core.Compiler.is_effect_free(effects)
+        @test Core.Compiler.is_nothrow(effects)
+        @test Core.Compiler.is_terminates(effects)
```

Comment:
More relevant for Base, but we should maybe have an interface for testing these "abnormal" effects more concisely, like the `Effects` constructor

Reply:
Do you mean we need something other than `!is_concrete_elibible` (`!is_total_or_error`, previously)?